src/services/graphql.ts

```diff
@@ -0,0 +1,199 @@
+            linjaByLintunnus(lintunnus:$lineId) {
+                ${lineQueryFields}
+                reittisByLintunnus(first: 1, orderBy: REIVIIMPVM_DESC) {
+                    nodes {
+                        reinimi
+                        reiviimpvm
+                    }
+                }
+            }
+        }`
+    );
+}
+
+function getAllLinesQuery() {
+    return (
+        gql`{
+            allLinjas {
+                nodes {
+                    ${lineQueryFields}
+                    reittisByLintunnus(orderBy: REIVIIMPVM_DESC) {
+                        nodes {
+                            reinimi
+                            reitunnus
+                            reiviimpvm
+                        }
+                    }
+                }
+            }
+        }`
+    );
+}
+
+function getRouteQuery() {
+    return (
+        gql`query getLineDetails($routeId: String!) {
+            route: reittiByReitunnus(reitunnus: $routeId) {
+                ${routeQueryFields}
+                reitinsuuntasByReitunnus{
+                    nodes {
+                        ${routePathQueryFields}
+                        reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta {
+                            nodes {
+                                ${routePathLinkQueryFields}
+                                solmuByLnkalkusolmu {
+                                    ${startNodeQueryFields}
+                                }
+                                solmuByLnkloppusolmu {
+                                    ${endNodeQueryFields}
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }`
+    );
+}
+
+function getRoutePathQuery() {
+    return (
+        gql`query getRoutePath($routeId: String!, $startDate: Datetime!, $direction: String!) {
+            routePath: reitinsuuntaByReitunnusAndSuuvoimastAndSuusuunta(reitunnus: $routeId, suuvoimast: $startDate, suusuunta: $direction) {
+                ${routePathQueryFields}
+                reittiByReitunnus {
+                    lintunnus
+                }
+                reitinlinkkisByReitunnusAndSuuvoimastAndSuusuunta {
+                    nodes {
+                        ${routePathLinkQueryFields}
+                        solmuByLnkalkusolmu {
+                            ${startNodeQueryFields}
+                        }
+                        solmuByLnkloppusolmu {
+                            ${endNodeQueryFields}
+                        }
+                    }
+                }
+            }
+        }`
+    );
+}
+
+function getLinksQuery() {
+    return (
+        gql`query getNodesWithRoutePathLinkStartNodeId($nodeId: String!) {
+            solmuBySoltunnus(soltunnus: $nodeId) {
+                linkkisByLnkalkusolmu {
+                    nodes {
+                        ${routeLinkQueryFields}
+                        geojson
```

Comment:
geojson is both declared here and in routeLinkQueryFields varible

Reply:
removed this